ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -8,6 +8,15 @@
 from ivy.functional.frontends.tensorflow.tensor import EagerTensor
 
 
+@to_ivy_arrays_and_back
+def argsort(values, axis=-1, direction='ASCENDING', stable=False, name=None):
+    if direction == 'DESCENDING':
+        descending = True
+    else:
+        descending = False
+    return ivy.argsort(values, axis=axis, descending=descending, stable=stable).astype("int32")
```

Comment:
Can you just fix this minor lint by putting the args on newline for each?

Reply:
Now the file is fixed